### PR TITLE
code cleanup and seach box added

### PR DESCRIPTION
**Pull request by vinodkumar45**

### CODE REVIEW

**Comment by vinodkumar45** [comment]
image

**Comment by vinodkumar45** [comment]
image

File: seeker-portal/src/App.css [hunk 1]
```diff
@@ -1,20 +1,65 @@
-  
+.tabs-list{
+  .MuiTabs-flexContainer{
+    background-color: #F3F4F5;
```
Comment (sri-pusuluri) [inline seeker-portal/src/App.css]: why r u not using color variables?

File: seeker-portal/src/App.css [hunk 2]
```diff
@@ -1,20 +1,65 @@
...
+    border-radius: 8px 8px 0 0;
```
Comment (sri-pusuluri) [inline seeker-portal/src/App.css]: px to rems

**Review by sri-pusuluri — APPROVED** [review]

**Review by sri-pusuluri — APPROVED** [review]

File: seeker-portal/src/App.css (hunk 1 above)
Comment (vinodkumar45) [reply seeker-portal/src/App.css]: Updated

**Pull request closed without merging**